inputFiles/poromechanics/PoroElastic_deadoil_3ph_baker_2d.xml

```diff
@@ -279,10 +279,12 @@
 
   <Outputs>
     <Silo
-      name="siloOutput"/>
+      name="siloOutput"
+      fieldNames="{ TotalDisplacement, pressure, skeleton_stress }"/>
```

Comment:
If possible, I would definitely implement a `from` `to` (some kind of a mapping).
That could be very useful for our users.

Reply:
@TotoGaz what kind of mapping do you suggest? Is there a case in which we want the names that appear in the vtk files to be different from the `fieldNames` in GEOSX?

Reply:
Oh, it's the output, not the input, stupid me 😵‍💫 
Forget what I said.